content/en/database_monitoring/setup_postgres/troubleshooting.md

```diff
@@ -179,17 +179,26 @@ Ensure that the Agent is running version 7.32.0 or newer. Datadog recommends reg
+#### Postgres extended query protocol
+
+If a client is using the Postgres [extended query protocol][13] or prepared statements then the Datadog Agent is not able to collect explain plans due to the separation of the parsed query and raw bind parameters. If the client provides an option to force using the simple query protocol then turning that on will enable the Datadog Agent to collect execution plans.
+
+| Language | Client                    | Configuration for simple query protocol                                                                                                                                                                                                                                                                                                                                                      |
+|----------|---------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Go       | [pgx][6]                  | Set `PreferSimpleProtocol` to switch to the simple query protocol ([Docs][14]).                                                                                                                                                                                                                                                                                                              |
+| Java     | [Postgres JDBC Client][15] | Set `preferQueryMode = simple` to switch to the simple query protocol ([Docs][16]).                                                                                                                                                                                                                                                                                                          |
+| Python   | [asyncpg][17]              | Uses the extended query protocol and it's not possible to disable it. Disabling prepared statements does not solve the problem.                                                                                                                                                                                                                                                              |
```

Comment:
Is there a reason why we don't have the pre-formatted statement workaround for `asyncpg` like we have for the others below? 
```
Format SQL Queries using [psycopg sql][17] before passing them to the DB client to enable collection of execution plans.
```

Reply:
Good question. I couldn't find a built-in sql formatter in `asyncpg`. We should still present a workaround though. I added some wording referring to the `psycopg` formatter.